cli/start.go

```diff
@@ -189,6 +189,9 @@ func runStart(_ *cobra.Command, _ []string) error {
 	}
 
 	info := build.GetInfo()
+	// We log build information to stdout (for the short summary), but also
+	// to stderr to coincide with the full logs.
+	log.Infof("[build] %s @ %s (%s)", info.Tag, info.Time, info.GoVersion)
```

Comment:
sure thing, done. I've also moved it back to the beginning. After trying a few different place, I found that there's something satisfying about having the first log line be the version string.